src/node_v8.cc

```diff
@@ -131,16 +153,19 @@ void BindingData::Deserialize(Local<Context> context,
                               int index,
                               InternalFieldInfoBase* info) {
   DCHECK_EQ(index, BaseObject::kEmbedderType);
-  HandleScope scope(context->GetIsolate());
+  v8::HandleScope scope(context->GetIsolate());
```

Comment:
Is this namespace prefix necessary?

```suggestion
  HandleScope scope(context->GetIsolate());
```